src/models/horizontal-scaling-summary/index.ts

```diff
@@ -5,6 +5,8 @@ export interface HorizontalScalingSummaryModel {
   maxReplicas: number;
   currentCPUUtilizationPercentage: number;
   targetCPUUtilizationPercentage: number;
+  currentMemoryUtilizationPercentage: number;
+  targetMemoryUtilizationPercentage: number;
```

Comment:
if the API now may omit these values, they should be marked as optional
```suggestion
  currentCPUUtilizationPercentage?: number;
  targetCPUUtilizationPercentage?: number;
  currentMemoryUtilizationPercentage?: number;
  targetMemoryUtilizationPercentage?: number;
```